webots_ros2_control/src/Ros2Control.cpp

```diff
@@ -40,6 +40,7 @@ namespace webots_ros2_control
   void Ros2Control::init(webots_ros2_driver::WebotsNode *node, std::unordered_map<std::string, std::string> &parameters)
```

Comment:
I think you can simply remove the word `parameters` on this line (leave the type definition).